repo_resource/common.py

```diff
@@ -218,7 +218,8 @@ def __init__(self, url, revision='HEAD', name='default.xml',
         self.__name = name
         self.__depth = depth
         self.__version: Version = None
-        self.__remote = {}
+        self.__rmt_url = {}
```

Comment:
Please don't do a refactor (variable name change) in a bugfix patch.

This makes things more difficult to review.

If you want to change `__remote` to `__rmt_url`, please do so in a prepatory patch.

Reply:
__remote had to be rename since we have now __remote_url and remote_rev
I'll do a separate patch

Reply:
ok. Let's make that `__remote_url`